From 5632b8ad22dbcaa4b6ea485bb852684125157f5f Mon Sep 17 00:00:00 2001
From: Manabu Matsui <manabu.matsui@gmail.com>
Date: Tue, 27 Mar 2018 16:27:24 +0900
Subject: [PATCH] [fix] Fix #467

Disown background timeout process started by __zplug::job::handle::hook()
---
 base/job/handle.zsh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/base/job/handle.zsh b/base/job/handle.zsh
index 88aea98..d304d42 100644
--- a/base/job/handle.zsh
+++ b/base/job/handle.zsh
@@ -263,7 +263,7 @@ __zplug::job::handle::hook()
                 builtin printf "$repo\n" >>|"$_zplug_build_log[rollback]"
             fi
             rm -f "$_zplug_lock[job]"
-        } &
+        } &!
     fi
 
     __zplug::utils::ansi::erace_current_line
